src/App.jsx

```diff
@@ -14,57 +16,60 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      {selectedGood === null ? (
+        <h1 className="title is-flex is-align-items-center">
+          No goods selected
+        </h1>
+      ) : (
+        <h1 className="title is-flex is-align-items-center">
+          {selectedGood} is selected
+          <button data-cy="ClearButton" type="button" className="delete ml-3" />
+        </h1>
+      )}
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+      <table className="table">
+        <tbody>
+          {goods.map(good => (
+            <tr data-cy="Good" key={good}>
```

Comment:
```suggestion
            <tr data-cy="Good" key={good} className={cn({ 'has-background-success-light': good === selectedGood })}>
```
has-background-success-light should be settled here.
use classnames library for dynamic classnames
```
import cn from 'classnames'
```

Reply:
> has-background-success-light should be settled here. use [classnames](https://www.npmjs.com/package/classnames) library for dynamic classnames
> 
> ```
> import cn from 'classnames'
> ```

that's next lesson